submit.py

```diff
@@ -306,6 +312,15 @@ def main():
         mainclass = opts.mainclass
     if opts.language:
         language = opts.language
+    else:
+        if language == 'Python':
+            try:
+                pyver = int(cfg.get('defaults', 'python-version'))
+                if pyver == 3:
+                    language = 'Python 3'
+            except:
+                pass
```

Comment:
It would be better to fail with an error message if the value in python-version is not 2 or 3.  I.e., only do the `cfg.get` inside the try-catch, don't cast to an `int`, but if `pyver not in ['2','3']` print an error message and quit.